docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,71 @@
+## Checklist
+<!--
+Mark an `[x]` for completed items.
+-->
+
+* [ ] Update CHANGELOG.rst for each commit separately:
```

Comment:
This could go under Commits and PR section

Reply:
@JureZmrzlikar , these items are meant to be a check list, therefore they are all the beginning under the Checklist section.

Reply:
For now I will leave only these 4 points as Checklist and the others as Guidelines. If we will notice this is not the optimal solution we can change it.